cypress/integration/testing.spec.ts

```diff
@@ -0,0 +1,33 @@
+describe('ngx-ui demo', () => {
+  before(() => {
+    cy.visit('/testing');
+    cy.get('.page-loader').should('not.exist', { timeout: 20000 });
+  });
+
+  it('getByLabel works with within', () => {
+    cy.getByLabel('Name').first().then(console.log).should('have.attr', 'id', 'input1');
```

Comment:
question: is there a reason to leave in the `console.log`?

Reply:
Nope... will delete.